spec/models/rating_spec.rb

```diff
@@ -19,9 +18,18 @@
   end
 
   it "is invalid if score is not between 0 - 4" do
-    rating = build(:rating, score: 5)
+    rating = build(:rating, score: 5, user: create(:user))
```

Comment:
I recently added `user` to the factory so this might not be necessary. I could be wrong on that with build though.